pandas/tests/indexes/datetimes/test_arithmetic.py

```diff
@@ -44,7 +45,74 @@ def addend(request):
 
 
 class TestDatetimeIndexComparisons(object):
-    # TODO: De-duplicate with test_comparisons_nat below
+    @pytest.mark.parametrize('other', [datetime(2016, 1, 1),
```

Comment:
can you move all of these comparisons (that you are moving anyway) to a new ``test_compare.py``

Reply:
Are you sure?  The other test_arithmetic.py files we've refactored out recently mostly include a TestComparison class.

Reply:
I would like to separate them, a followup is ok.